charts/rancher-logging/0.1.2/charts/fluentd/charts/fluentd-linux/templates/daemonset.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: {{ template "daemonset_api_version" . }}
 kind: DaemonSet
 metadata:
-  name: {{ template "fluentd.fullname" . }}
+  name: {{ template "fluentd.fullname" . }}-linux
```

Comment:
What is the upgrade impact of change the name of this daemonSet?

Reply:
@cjellick  Because we separate to Linux and Windows, so we add a suffix for that Daemonset, we use force upgrade which will remove the old chart and install the new chart. Because some changes between two version charts may cause upgrade failed, so we use force upgrade, for example, we remove label chart under spec.selector.matchLabels, and matchLabels is an unmodifiable field in apps/v1 Daemonset.